workspace/__lib__/xod/color-pattern/project.xod

```diff
@@ -0,0 +1,6 @@
+{
+  "description": "The color pattern type is a Linked List of colors. This type might be used by WS2812 for lighting up a pattern on LED matrix and etc.",
+  "license": "AGPL-3.0",
+  "name": "color-pattern",
+  "version": "0.1.0"
+}
```

Comment:
The pattern is soo specific to WS2812 now, that it would be better to merge it to `xod-dev/ws2812`. If this would be usable with other devices (which?), we can extract it back.